R/recode-with-table.R

```diff
@@ -166,15 +166,15 @@ rec_with_table <-
       source(custom_function_path)
     }
     if (is.null(variable_details)) {
-      print("Loading cchsflow variable_details")
+      message("Loading cchsflow variable_details")
```

Comment:
Should we suppress these messages if print_notes = FALSE?

Reply:
I think we should change `print_notes` to `notes`. Technically, we aren't printing — so that is a bit confusing and probably first raised the flag with CRAN checks. Also a bit verbose.

Reply:
These aren't notes they are similar to attaching a package and it telling u what is attached alongside.

Reply:
Since changing `print_notes` to `notes` is a breaking change that would require modifications to many files, should we hold off on this change for later releases?

Reply:
Well, shouldn't we do it for those same reasons?